### ISSUE TITLE
Gas Extractor

### ISSUE THREAD

**Issue by c0mm0n** [matt66146/FarmLog#2]
Hey, getting motes with Zapthrottle Mote Extractor isnt accounted by FL.

Great addon

**Comment by matt66146** [comment]
Do you have the Misc option enabled under the settings? The mote extractor was added under that setting (due to how the mote extractor loot event works, there are some other events like loot from the mail that will also popup). It is not enabled by default because of this.

**Comment by c0mm0n** [comment]
Was not, let me try again.
Ty for super fast answer

**Comment by c0mm0n** [comment]
<img width="307" alt="image" src="https://user-images.githubusercontent.com/1181493/124332114-fe1cb580-db90-11eb-975c-ee030f062264.png">

Works thx!
